src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1945,43 +1945,71 @@ public abstract partial class DbDataReader : System.MarshalByRefObject, System.C
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
         public abstract bool GetBoolean(int ordinal);
+        public bool GetBoolean(string name) => throw null;
         public abstract byte GetByte(int ordinal);
+        public byte GetByte(string name) => throw null;
         public abstract long GetBytes(int ordinal, long dataOffset, byte[] buffer, int bufferOffset, int length);
+        public long GetBytes(string name, long dataOffset, byte[] buffer, int bufferOffset, int length) => throw null;
         public abstract char GetChar(int ordinal);
+        public char GetChar(string name) => throw null;
         public abstract long GetChars(int ordinal, long dataOffset, char[] buffer, int bufferOffset, int length);
+        public long GetChars(string name, long dataOffset, char[] buffer, int bufferOffset, int length) => throw null;
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
         public System.Data.Common.DbDataReader GetData(int ordinal) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public System.Data.Common.DbDataReader GetData(string name) { throw null; }
```

Comment:
Nitpick: Use `{ throw null; }` or `=> throw null;` consistently in this file; don't use both syntaxes.

Reply:
Agree. In my defence though, it was already inconsistent 😄

Anyway, Change it eveywhere in `DbDataReader` to `=> throw null;`.